index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css"
+      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+      crossorigin="anonymous"
+    />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="main">
+      <div class="header">
+        <div class="container">
+          <div id="title">
+            <h1>BOB <span> MARLEY</h1>
```

Comment:
Please make the logo <a> tag and assign it to a page (home page). This will improve the user experience, and make the user traverse through the pages easily.

Reply:
- [OPTIONAL ] Great job @felixDev22 , I think this should more interested if you use also some emojis for being so human friend and this encourage the student to make your request changes :wink:

Reply:
Thank you...will surely make use of emojis 😄 👍